src/views/templates/details/tabs/scheduling/components/Descheduler.tsx

```diff
@@ -58,22 +63,33 @@ const Descheduler: React.FC<TemplateSchedulingGridProps> = ({ template }) => {
       </DescriptionListTermHelpText>
 
       <DescriptionListDescription>
-        {useDeschedulerInstalled && (
-          <Button
-            isInline
-            isDisabled={!isDeschedulerInstalled}
-            onClick={() =>
-              createModal(({ isOpen, onClose }) => (
-                <DeschedulerModal template={template} isOpen={isOpen} onClose={onClose} />
-              ))
-            }
-            variant="link"
-            iconPosition={'right'}
-          >
-            {isDeschedulerOn ? t('ON') : t('OFF')}
-            <PencilAltIcon className="co-icon-space-l pf-c-button-icon--plain" />
-          </Button>
-        )}
+        <Tooltip
+          content={
+            !isDeschedulerInstalled &&
+            t(
+              'To enable the descheduler, you must install the Kube Descheduler Operator from OperatorHub and enable one or more descheduler profiles.',
+            )
+          }
+          position="right"
+        >
+          <span>
+            <Button
+              isInline
+              isDisabled={!isDeschedulerInstalled}
+              onClick={() =>
+                createModal(({ isOpen, onClose }) => (
+                  <DeschedulerModal template={template} isOpen={isOpen} onClose={onClose} />
+                ))
+              }
+              variant="link"
+              iconPosition={'right'}
+            >
+              {isDeschedulerInstalled && isDeschedulerOn ? t('ON') : t('OFF')}
```

Comment:
if the vm has Descheduler annotation set to on, a user should see that it is on

for example:
if a user get from cluster admin a template that has Descheduler on,
then check in the UI and see that it's off
this is not the correct behavior

we can do something like:
a - no descheduler annotations => 'OFF'
b - deschedualer annotation + operator installed => 'ON'
c - deschedualer annotation + operator not installed => 'ON (deschedualer operator not installed, learn more ^ )' with a learn more link to kubevirt doc about deschedualer

Reply:
p.s.
did you check that non-privilage user (e.g. test user) can watch operator,
can a non-privlage user know if the operator installed or not ?

Reply:
The descheduler has to be installed, only then it makes sense to look at the annotation. I was told to implement it so also in our older repo... Also in the design doc, there is:
_If the operator isn’t installed, the Descheduler is disabled as well._
+ the picture with descheduler set to OFF

Reply:
> did you check that non-privilage user (e.g. test user) can watch operator,
can a non-privlage user know if the operator installed or not ?

No, because I was told not to do too many things in one PR. Also I think we are already mixing too much stuff in this PR. This PR is not about the Descheduler itself, it's just about adding the missing tooltip, according to the design. The discussion about how the Descheduler should work, should have been done here: https://github.com/kubevirt-ui/kubevirt-plugin/pull/348 If it is buggy, I suggest to make sure what is buggy or which design doc to follow, and then to make another PR especially for the purpose of fixing the Descheduler's bugs.